linux_os/guide/system/logging/ensure_rtc_utc_configuration/ansible/shared.yml

```diff
@@ -5,8 +5,8 @@
     timedatectl status | grep -i 'Time zone'| grep -iv 'UTC\|GMT' || true
   register: check_tz
   failed_when: "check_tz.rc not in [ 0 , 1 ]"
- 
+
 - name: Configure OS to use 'UTC' timezone
-  command: timedatectl set-timezone UTC 
+  command: timedatectl set-timezone UTC
   become: yes
-  when: check_tz.rc == 0
+  when: check_tz is not skipped and check_tz.rc == 0
```

Comment:
I don't think it is relevant to include this new criterion in this Playbook. It won't have any effect. And considering there is already a treatment to fail the playbook, I believe the changes are not necessary.